src/Evolution/DgSubcell/Tags/Jacobians.hpp

```diff
@@ -0,0 +1,36 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace evolution::dg::subcell::fd::Tags {
+/// \brief The inverse Jacobian from the logical frame to the grid frame at the
+/// cell centers.
+///
+/// Specifically, \f$\partial x^{\bar{i}} / \partial x^i\f$, where \f$\bar{i}\f$
+/// denotes the logical frame and \f$i\f$ denotes the grid frame.
+///
+/// \note stored as a `std::optional` so we can reset it when switching to DG
+/// and reduce the memory footprint.
+template <size_t Dim>
+struct InverseJacobianLogicalToGrid : db::SimpleTag {
```

Comment:
The `domain::Tags::InverseJacobian` is templated on source- and target-frame, you could do that here as well instead of hard-coding the frames, to support Jacobians in other frames?

Question: Do you need the separate tag (from the one in `domain`) because the Jacobian is evaluated on the FD grid, as opposed to the spectral collocation points?

Reply:
Yes, I could, but currently I don't expect I'll need more for caching (inertial frame is always time-dependent anyway). If the need arises it'll be fairly easy to switch.

Yes, that's right. And it's not easy to reuse those tags because basically everything in `Domain` assumes one grid per element, not 2. I'd also like to avoid having too much memory allocated for the subcell solver while it's not in use, since that'll limit how few cores we can run on (I'd rather not have the requirement of "you need to run on at least 50 nodes...")